packages/core/test/insured-bridge/helpers/SmockitHelper.js

```diff
@@ -0,0 +1,18 @@
+const { getContractDefinition } = require("@eth-optimism/contracts");
```

Comment:
Random question: why isn't there a github repo linked to this npm package? All the contract file paths such as `@eth-optimism/contracts/libraries/bridge/OVM_CrossDomainEnabled.sol` don't exist in what I thought was the optimism [contracts repo](https://github.com/ethereum-optimism/optimism/tree/develop/packages/contracts)

Reply:
no idea 🤷